tests/users/test_views.py

```diff
@@ -57,6 +58,9 @@ def test_logout_with_next(user, client, logout_url):
 
 
 @pytest.mark.django_db
+@override_settings(
+    SENDINBLUE_API_KEY=None
+)
```

Comment:
Fixing the test, it's easy to see, that we should probably add sth. to test that the settings are also set properly, not only if they are set. But I think, we can do that in a second iteration together with adding tests.